packages/experiments/src/components/Shimmer/examples/Shimmer.LoadData.Example.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { Shimmer } from 'experiments/lib/Shimmer';
+import { Shimmer } from '../Shimmer';
```

Comment:
this should be `@uifabric/experiments/lib/Shimmer`

Reply:
I might be wrong.

Reply:
@dzearing This is in the same package and linter is complaining about it. Why not `../Shimmer`?

Reply:
The build is failing if I use `@uifabric/experiments/lib/Shimmer`

Reply:
```
ERROR in ./src/components/Shimmer/examples/Shimmer.Application.Example.tsx
Module not found: Error: Can't resolve '@uifabric/experiments/lib/Shimmer' in '/Users/vitaliebraga/Desktop/UIFabric/office-ui-fabric-react/packages/experiments/src/components/Shimmer/examples'
 @ ./src/components/Shimmer/examples/Shimmer.Application.Example.tsx 11:16-60
 @ ./src/components/Shimmer/ShimmerPage.tsx
 @ ./src/demo/AppDefinition.tsx
 @ ./src/demo/index.tsx
 @ multi (webpack)-dev-server/client?http://localhost:4322 ./src/demo/index.tsx
ℹ ｢wdm｣: Failed to compile.
```

Reply:
This is a code example. A user reading it can't use your example when you are using relative paths in examples :)

Reply:
The project webpack config and tsconfig are messed up here and should be resolving `@uifabric/experiments`. 
I'm ok checking this in, but we need to scrub these examples and the configs